decidim-core/app/views/layouts/decidim/_timeout_modal.html.erb

```diff
@@ -1,9 +1,9 @@
-<% if current_user %>
-  <% timeout_time_seconds = Devise.timeout_in.to_i %>
+<% if current_user && !current_user.remember_created_at %>
+  <% timeout_time_seconds = Decidim.expire_session_after.to_i %>
   <div class="reveal" id="timeoutModal" data-close-on-click="false" data-close-on-esc="false"
     data-seconds-until-timeout-path="<%= decidim.seconds_until_timeout_path %>"
     data-session-timeout="<%= timeout_time_seconds %>"
-    data-session-timeouter-interval="<%= Decidim.config.session_timeouter_interval %>" data-reveal>
+    data-session-timeout-interval="<%= Decidim.config.session_timeout_interval %>" data-reveal>
```

Comment:
This needs to be also modified after changing the configuration option to duration:

```erb
data-session-timeout-interval="<%= Decidim.config.session_timeout_interval.in_seconds * 1000 %>" 
```